index.html

```diff
@@ -83,9 +84,9 @@ <h3 class="second_project_title">Profesional Art Printing Data</h3>
       <li class="second_project_skills flex">Ruby</li>
     </ul>
     <button class="see_project_works2">See Project</button>
-  </div>
+   </div>
 
-  <div class="third_project">
+   <div class="third_project">
     <h3 class="third_project_title">Profesional Art Printing Data</h3>
     <p class="third_project_content">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
     <ul class="third_project_skill_list flex">
```

Comment:
- [ ] I would suggest please adjust the spacing on the left and right of the page. Also, stick to the design as much as possible. please fix the background and color of the button.
- [ ] please implement in all the cards
### Your design
![yourrrrr](https://user-images.githubusercontent.com/81727988/206659893-69260cb9-28c8-4a28-87ac-f99c40d00739.png)

### Figma design
![fogma](https://user-images.githubusercontent.com/81727988/206659915-a7162dcf-9fdd-4347-bc1f-1935f28252aa.png)